cloud-formation/src/state-machine.yaml

```diff
@@ -12,6 +12,10 @@ States:
   CreateZuoraSubscription:
     Type: Task
     Resource: "$${CreateZuoraSubscriptionLambda.Arn}"
+    Next: SendThankYouEmail
+  SendThankYouEmail:
+    Type: Task
+    Resource: "$${SendThankYouEmailLambda.Arn}"
     Next: FlakeyService
```

Comment:
We should probably get rid of the FlakeyService lambda now! Happy to do that in another PR though